build-image/README.md

```diff
@@ -1,2 +1,58 @@
 # Verifiable build using Docker
 
+Docker image based on our base CI image `<base-ci-linux:latest>`.
+
+Used for reproducible builds in `cargo contract --verifiable`
+
+## Dependencies and Tools
+
+- `llvm-dev`
+- `zlib1g-dev`
+- `npm`
+- `yarn`
+- `wabt`
+- `binaryen`
+
+**Inherited from `<base-ci-linux:latest>`**
+
+- `libssl-dev`
+- `clang-10`
+- `lld`
+- `libclang-dev`
+- `make`
+- `cmake`
+- `git`
+- `pkg-config`
+- `curl`
+- `time`
+- `rhash`
+- `ca-certificates`
+- `jq`
+
+**Rust versions:**
+
+Currently, the 1.69 toolchain is temporarily required to build ink! contracts because of https://github.com/paritytech/cargo-contract/issues/1139
+
+**Rust tools & toolchains:**
+
+We use stable releases from crates.io
+
+- `cargo-contract`
+- `cargo-dylint` and `dylint-link`
+- `pwasm-utils-cli`
+- `solang`
+- `wasm32-unknown-unknown`: The toolchain to compile Rust codebases for Wasm.
+
+[Click here](https://hub.docker.com/repository/docker/paritytech/contracts-ci-linux) for the registry.
+
+## Usage
+
+The default entry point is `ENTRYPOINT [ "cargo", "contract", "build", "--release" ]`
+and work directory is set to `/contract`. You need to mount the folder with the contract to the container.
+
+```bash
+docker run -d \
+    --name ink-container \
+    --mount type=bind,source="$(pwd)",target="/contract" \
+    paritytech/contracts-verified
```

Comment:
Worth mentioning that this won't quite work if you have path-dependencies on other contracts in `Cargo.toml`, since the mounted volume will see only the _current_ contract (dir) and not things that are "next to" (or anywhere else). For example:
```sh
- /foo
- - lib.rs
- - Cargo.toml
- /bar
- - lib.rs
- - Cargo.toml
```
and in `foo/Cargo.toml` you have
```toml
[dependencies]
bar = { path = "../bar", features = ["ink-as-dependency"]
```

In this case, you need to run the verification in folder common to both and use `--manifest-path foo/Cargo.toml`.

Reply:
This will work if you point root dir of 2 contracts to `/contract` and specify `--manifest-path <relative/dir>` as you correctly said.

Reply:
Yes, it's just that's not what this README say - that's all I'm saying. The `ENTRYPOINT` for the docker is already defined so if you call it w/o overriding then it won't work. This won't be obvious to people that use the tool since the non-verified version will work correctly w/o any additional args.

Reply:
Good point. I'll update the README to include instructions for multi-contract projects.

Reply:
Ideally, this tool would detect types of dependencies in the contact, recognize where there are path-dependent ones and then mount them automatically - without the need for manual intervention. That way it's completely opaque to the user and the experience is unified.

Reply:
Can do as a follow-up issue - to detect path dependencies for other contracts and automatically mount the relevant dirs to the contract.

Let's roll out the core functionality for now.